### PR TITLE
Updated availability information

### PR DESCRIPTION
**Pull request by anwara-ms**
We rolled out the feature in GCC and updated some timelines.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/information-barriers-in-teams.md [hunk 1]
```diff
@@ -204,3 +204,7 @@ For more information on licenses and permissions, including plans and pricing, s
 - To set up IB policies, see [Define policies for information barriers](https://docs.microsoft.com/office365/securitycompliance/information-barriers-policies).
 
 - To edit or remove IB policies, see [Edit (or remove) information barrier policies](https://docs.microsoft.com/microsoft-365/compliance/information-barriers-edit-segments-policies).
+
+## Availability
+- The feature is available in our public cloud, in Jan 2021, we rolled out Information Barriers in GCC cloud.
```
Comment (JohanFreelancer9) [inline Teams/information-barriers-in-teams.md:209]:
```suggestion
- The feature is available in our public cloud; in January 2021, we rolled out Information Barriers in the GCC cloud.
```

File: Teams/information-barriers-in-teams.md [hunk 2]
```diff
@@ -204,3 +204,7 @@ For more information on licenses and permissions, including plans and pricing, s
...
+- The feature is still not yet available in GCCH and DOD cloud
```
Comment (JohanFreelancer9) [inline Teams/information-barriers-in-teams.md:210]:
```suggestion
- The feature is not yet available in the GCCH and DOD clouds.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@chrfox PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by chrfox** [comment]
@robmazz owns this space now, re-assigning to him

**Pull request merged**